RecoLocalMuon/CSCRecHitD/python/cscRecHitD_cfi.py

```diff
@@ -4,73 +4,73 @@
 
 # parameters for CSC rechit building
 from RecoLocalMuon.CSCRecHitD.cscRecHitD_cff import *
-csc2DRecHits = cms.EDProducer("CSCRecHitDProducer",
+import RecoLocalMuon.CSCRecHitD.cscRecHitDProducer_cfi as _mod
+
+csc2DRecHits = _mod.cscRecHitDProducer.clone(
     #
     #    Parameters for coordinate and uncertainty calculations
```

Comment:
These comment lines must remain attached to the `cscRecHitDParameters` , now moved at the end of this file